db/migrate/20220907161526_add_status_to_policy_classes.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class AddStatusToPolicyClasses < ActiveRecord::Migration[6.1]
+  def up
+    add_column :policy_classes, :status, :integer
+
+    execute(
+      "UPDATE policy_classes
+      SET status = 0
+      FROM policies
+      WHERE policies.policy_class_id = policy_classes.id
+      AND policies.status = 2;"
+    )
+
+    execute(
+      "UPDATE policy_classes
+      SET status = 1
+      WHERE status IS NULL;"
+    )
```

Comment:
If a policy class has any policies that are 'to be determined' set the status as 'in assessment'. Otherwise set the status as 'complete'.